src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -584,54 +582,26 @@ def add_electrical_series(
         warn(f"compression_opts ({compression_opts}) were passed, but compression type is 'lzf'! Ignoring options.")
         compression_opts = None
 
+    default_name = f"ElectricalSeries_{write_as}"
```

Comment:
@h-mayorquin We added a best practice a while back that naming conventions of classes ought to follow CamelCase (we also tend to use it ourselves whenever possible). This mixture of CamcelCase and snake_case (which even goes against ADA 95 suggestions for combining the two) has always looked a bit awkward - how would you feel about adjusting this to `f"ElectricalSeries{write_as.capitalize()}"`?

Reply:
Yes, sure. I think it makes sense. We should uphold our own conventions whenever possible.